kep/9-capacity-scheduling/README.md

```diff
@@ -320,6 +321,233 @@ preemptor’s priority as potential victims in a node.
+  namespace: quota2
+spec:
+  max:
+    cpu: 2
+  min:
+    cpu: 0
+```
+##### Elastic Quota Configuration for namespace 3
+```
+apiVersion: v1
+kind: ElasticQuota
+metadata:
+  name: quota3
+  namespace: quota3
+spec:
+  max:
+    cpu: 2
+  min:
+    cpu: 1
+```
+#### Deployment on Namespace quota1
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: quota1
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      name: nginx
+      labels:
+        app: nginx
+    spec:
+      schedulerName: capacityscheduler
+      containers:
+        - name: nginx
+          image: nginx
+          resources:
+            limits:
+              cpu: 1
+            `requests:`
+              cpu: 1
+```
+The nginx pod is able to run because it will borrow the 1 free cpu min from namespace quota3. Note, by default if the PriorityClassName is not configured, the pod will have a priority of 0.
+
+#### A sample Priority Class Deployment
+```
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: high-priority
+value: 1000000
+globalDefault: false
+description: "Sample High Priority Class"
+```
+
+#### Deployment on Namespace quota2
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: quota2
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      name: nginx
+      labels:
+        app: nginx
+    spec:
+      schedulerName: capacityscheduler
+      priorityClassName: high-priority
+      containers:
+        - name: nginx
+          image: nginx
+          resources:
+            limits:
+              cpu: 1
+            `requests:`
+              cpu: 1
+```
+In this case, the high priority pod in namespace quota2 cannot run even though it has a higher priority compared to the running pod in namespace quota1 because the capacity scheduler follows first come first serve rule for pods overusing their namespace resource min. 
+
+More specifically, during the PostFilter plugin, the scheduler will first determine if running the current pod will lead to the exceed of the resource min in its current namespace. If it will, the scheduler can **only preempts pods that have lower priority in the same namespace*** but not pods from other namespaces because of the first come first serve rule.
+
+When the high priority pod in namespace quota2 is deployed, since there are no more unused min resources in the elastic quota namespaces, the scheduler will start the preemption process. 
+
+During the preemption process, since namespace quota2 after running this high priority pod will overuse its namespace resource min, the capacity scheduler can only preempt lower priority pods in the same namespace (namespace quota2). Since currently there is no other pods the scheduler can preempt in namespace quota2, the high priority pod fails to get scheduled.
```

Comment:
I think the previous explanation is clear enough, so prefer to remove these stanzas.